packages/components/input-number/__tests__/input-number.test.tsx

```diff
@@ -2,124 +2,76 @@ import { nextTick, ref } from 'vue'
 import { mount } from '@vue/test-utils'
 import { describe, expect, it, test } from 'vitest'
 import { ArrowDown, ArrowUp } from '@element-plus/icons-vue'
-import { ElFormItem } from '@element-plus/components/form'
+import { ElFormItem as FormItem } from '@element-plus/components/form'
```

Comment:
```suggestion
import { ElFormItem } from '@element-plus/components/form'
```

use `ElFormItem` directly.

Reply:
resolved.